code/game/gamemodes/blob/theblob.dm

```diff
@@ -24,6 +24,7 @@
 	M.changeNext_move(CLICK_CD_MELEE)
 	var/a = pick("gently stroke", "nuzzle", "affectionatly pet", "cuddle")
 	M.visible_message("<span class='notice'>[M] [a]s [src]!</span>", "<span class='notice'>You [a] [src]!</span>")
+	to_chat(overmind, "<span class='notice'>[M] [a]s you!</span>")
```

Comment:
it has a click delay, and also the blob is perfectly capable of just killing them, since they need to be next to the blob.